app/models/custom_filter.rb

```diff
@@ -24,6 +24,29 @@ class CustomFilter < ApplicationRecord
   enum filter_type: { conversation: 0, contact: 1, report: 2 }
   validate :validate_number_of_filters
 
+  def records_count
+    fetch_record_count_from_redis
```

Comment:
I think this should have a fallback, in case the record count does not exist in redis, it should calculate, save it in redis and return it. 
CC: @sojan-official what do you think?

Reply:
yep . ref: https://github.com/chatwoot/chatwoot/pull/7164#discussion_r1205545166 

- lets requeue the job in case count is nil

Reply:
@sojan-official @scmmishra I have added these changes please have a look.